src/index.mjs

```diff
@@ -18,7 +18,7 @@ export const and = Pkg.and;
 export const or = Pkg.or;
 
 // export * from './lib/query-interface';
-export const QueryInterface = Pkg.QueryInterface;
+export const AbstractQueryInterface = Pkg.QueryInterface;
```

Comment:
```suggestion
export const AbstractQueryInterface = Pkg.AbstractQueryInterface;
```

Reply:
I see - I need to give more of a look at how dual-mode exports work. Seems like at a high level the `index.d.ts` file has to match `index.mjs` though?

Reply:
`Sequelize.AbstractQueryInterface` is the export of the class, and it's added as a static property on the Sequelize class for legacy reasons, which is that the Sequelize class is the default export:

```ts
// this works
const Sequelize = require('@sequelize/core');

// therefore, doing this gets the "AbstractQueryInterface" static property from the Sequelize class
const { AbstractQueryInterface } = require('@sequelize/core');

// This also works and is the preferred way of importing Sequelize.
const { Sequelize } = require('@sequelize/core');
```

`queryInterface: AbstractQueryInterface` is an instance property on the Sequelize class. It's not an export but the current instance of QueryInterface

Reply:
Makes sense - thank you!